web3/contract.py

```diff
@@ -243,7 +243,7 @@ def __init__(self, abi: ABI, web3: 'Web3', address: ChecksumAddress=None) -> Non
                         address=address,
                         event_name=event['name']))
 
-    def __getattr__(self, event_name: str) -> "ContractEvent":
+    def __getattr__(self, event_name: str) -> Type['ContractEvent']:
```

Comment:
I had copied, but not yet pasted the class code, when my intuition told me to leave these as strings, as to avoid location-dependent code in a nearly 1700-line file. Not strongly opposed to shuffling the order, but my intuition is also telling me this will not be the last change required when additional types are suddenly out of order/not declared in time.